avaframe/com1DFA/com1DFA.py

```diff
@@ -1762,6 +1768,7 @@ def DFAIterate(cfg, particles, fields, dem, inputSimLines, outDir, cuSimName, si
     massEntrained = []
     massDetrained = []
     massTotal = []
+    pfvTimeMax = []
```

Comment:
What is the purpose of this variable? And the difference to pfv?

Reply:
it is an array of the max value for each time step - needed for the new mass plot better naming would be fvTime Max so it is a scalar max value over all cells but for each time step